benchmarks/IntegerBenchmark.php

```
@@ -0,0 +1,180 @@
+    }
+
+    public function benchmarkBigIntegerOperations(): array
+    {
+        $int1 = new Int64('9223372036854775800');
+        $int2 = new Int64('7');
+
+        $start = microtime(true);
+        $memoryStart = memory_get_usage();
+
+        for ($i = 0; $i < self::ITERATIONS / 100; $i++) { // Fewer iterations for big ints
+            $result = $int1->add($int2);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)